Makefile

```diff
@@ -0,0 +1,17 @@
+.PHONY: piptools upgrade
+
+COMMON_CONSTRAINTS_TXT=requirements/common_constraints.txt
+.PHONY: $(COMMON_CONSTRAINTS_TXT)
+$(COMMON_CONSTRAINTS_TXT):
```

Comment:
I think this is an old pattern we want to move away from,  you can just reference the common_constraints.txt file directly in your local constranint file.

Reply:
I've removed the pattern